src/backend_api.ts

```diff
@@ -0,0 +1,69 @@
+import { Config } from '@backstage/config';
+import { createApiRef } from '@backstage/core-plugin-api';
+import { AutoSyncConfiguration, OpsLevelBackendApi } from './types/OpsLevelData';
+
+export const opslevelPluginApiRef = createApiRef<OpsLevelBackendApi>({
+  id: 'plugin.opslevel.backend.service',
+});
+
+export class OpsLevelPluginAPI implements OpsLevelBackendApi {
+  static fromConfig(config: Config) {
+    return new OpsLevelPluginAPI(config.getString('backend.baseUrl'));
+  }
+
+  constructor(public url: string) {
+    this.url = `${this.url}/api/opslevel`;
+  }
+
+  public async isPluginAvailable() {
+    return await fetch(`${this.url}/ping`)
+      .then((response) => response.json())
+      .then((data) => {
+        return data.status === "OpsLevel Backend Plugin is present";
+      })
+      .catch(() => {
+        return false;
+      });
+  }
```

Comment:
Mixing `await` with promise chaining is an antipattern. Try sticking to just one convention:

```suggestion
  public async isPluginAvailable() {
    try {
      const response = await fetch(`${this.url}/ping`);
      return response.json().status === "OpsLevel Backend Plugin is present";
    } catch {
      return false;
  }
```

Reply:
nit: can probably just check the status of the _request_:

```javascript
    return response.status === 200;
```